### ISSUE TITLE
App Crashes on notebook edit if name contains a slash

### ISSUE THREAD

**Issue by sgpthomas** [Philip-Scott/Notes-up#49]
It's all in the title. But here's the boring terminal output in case you need it.

```bash
** (Notes:14818): ERROR **: Notebook.vala:66: Error renaming directory: File names cannot contain '/'
Trace/breakpoint trap (core dumped)
```

**Comment by Philip-Scott** [comment]
The boring terminal output actually helped to see what's wrong :smiley: 

Yeah, since each notebook internally acts like a folder, you can't have that character in it.... I'll make it so you can't access those illegal characters. Thank you for reporting it! :)

**Comment by sgpthomas** [comment]
Haha I know thats why I gave it to you :)

**Comment by Philip-Scott** [comment]
It's being fixed on the Latest commit e873161 :)
